examples/prom_02_grafana_cloud/prom_02_grafana_cloud.ino

```diff
@@ -16,10 +16,10 @@ WriteRequest req(2, 1024);
 // See the README at https://github.com/grafana/prometheus-arduino for more info.
 
 // Define a TimeSeries which can hold up to 5 samples, has a name of `uptime_milliseconds`
-TimeSeries ts1(5, "uptime_milliseconds_total", "{job=\"esp32-test\",host=\"esp32\"}");
+TimeSeries ts1(5, (char*)"uptime_milliseconds_total", (char*)"{job=\"esp32-test\",host=\"esp32\"}");
```

Comment:
Rather than having to force casts I should be able to make the function params `const char*` can you remove these casts for now and I'll fix that in a separate PR?

Reply:
Will do